src/libcharon/sa/ikev2/tasks/ike_rekey.c

```diff
@@ -137,6 +137,8 @@ static void establish_new(private_ike_rekey_t *this)
 		{
 			lib->processor->queue_job(lib->processor, job);
 		}
+		charon->kernel->write_terminator_tunnel_ike_spis(charon->kernel, this->new_sa);
```

Comment:
Again this will not compile in client build. Needs to be under `#ifdef USE_KERNEL_SUPER`